### ISSUE TITLE
can you stop chaquopy

### ISSUE THREAD

**Issue by 5tiaowu** [chaquo/chaquopy#933]
Can you close chaquopy, it will keep running after startup, so if I have other compiled chaquopy folders, I can’t run it, and I can run the first one. If I can stop it, I can load others

**Comment by mhsmith** [comment]
Sorry, I don't understand your question. Please give a specific example.

**Comment by 5tiaowu** [comment]
A process can only run one chaquopy. For example, I compiled two chaquopy folders and put them in assets. I started the first one and the second one couldn’t start because a process can only run one, so I want to stop chaquopy and run new python codes. That is, can the chaquopy folder be placed locally? Not in assets, because hot updates are required every time the chaquopy folder is updated, which consumes a lot of traffic.

**Comment by 5tiaowu** [comment]
> Sorry, I don't understand your question. Please give a specific example.

A process can only run one chaquopy. For example, I compiled two chaquopy folders and put them in assets. I started the first one and the second one couldn’t start because a process can only run one, so I want to stop chaquopy and run new python codes. That is, can the chaquopy folder be placed locally? Not in assets, because hot updates are required every time the chaquopy folder is updated, which consumes a lot of traffic.
